source/refunding_payments/index.html.md.erb

```diff
@@ -52,7 +52,7 @@ You can see definitions, values, and the limits of every parameter and response
 |Status|Meaning|
 |---|---|
 |pending|You cannot refund the payment yet because your user has not completed the payment.|
-|unavailable|You cannot refund the payment, usually because the payment failed.|
+|unavailable|You cannot refund the payment. There are two possible reasons for this:<br><br><li>the payment failed<li>the payment was processed by a PSP that we no longer support, such as ePDQ or Smartpay|
```

Comment:
I think there’s actually more than two reasons. I think if a payment has already been fully refunded the status will be `unavailable`.

Reply:
@alexbishop1 I've just tested it and a fully refunded payment returns `"status": "full"`

Reply:
Fair enough.